.gitignore

```
@@ -3,3 +3,5 @@
 !/logs/example/
 node_modules/
 .vscode/*
+algo-core/__pycache__/*
```

A nitpick but I'd say a standard (and reusable) way of ignoring the cache is just
```suggestion
__pycache__/
```

---

yeah, that was lazy me using the VSCode to add it for me instead of typing it in the file